### PR TITLE
Implementing Searching feature for air_quality_data table 

### PR DESCRIPTION
**Pull request by quandang246**
## Related Tickets
- [[ticket redmine](https://edu-redmine.sun-asterisk.vn/projects/hn-python-naitei19-air-quality-management)](https://edu-redmine.sun-asterisk.vn/issues/67634)

## WHAT (optional)
- Implementing Searching feature for air_quality_data table 

## HOW
- I edit homepge.html file and adding js part. 

## WHY (optional)
- 

## Evidence (Screenshot or Video)
image
image

### CODE REVIEW

**Review by tdanhjv — APPROVED** [review]

**Comment by tdanhjv** [comment]
確認した

**Review by riderhuy — APPROVED** [review]
Approve

**Comment by quandang246** [comment]
ready

File: air_quality_management/myapp/templates/homepage.html [hunk 1]
```diff
@@ -31,4 +32,20 @@ <h1 style="color: white; padding: 0 0 20px 0;">{% trans "AIR QUALITY MANAGEMENT
...
+          for (var i = 0; i < tableRows.length; i++) {
+            var row = tableRows[i];
+            if (!row.textContent.toLowerCase().includes(searchInput.value.toLowerCase())) {
+              row.style.display = 'none';
+            } else {
+              row.style.display = '';
+            }
+          }
+        });
+      </script>
```
Comment (huongnt-2545) [inline air_quality_management/myapp/templates/homepage.html]: move file js e nhé, ko viết js ở template

File: air_quality_management/myapp/templates/homepage.html [hunk 2]
```diff
@@ -6,6 +6,7 @@
 {% block content %}
     <div class="container-1">
         <h1 style="color: white; padding: 0 0 20px 0;">{% trans "AIR QUALITY MANAGEMENT FOR CITIES" %}</h1>
+        <input type="text" id="searchInput" placeholder="{% trans 'Search for locations...' %}">
```
Comment (huongnt-2545) [inline air_quality_management/myapp/templates/homepage.html:10]: chạy lệnh make + compile message

**Comment by quandang246** [comment]
ready

File: air_quality_management/myapp/static/js/search.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+        for (var i = 0; i < tableRows.length; i++) {
+            var row = tableRows[i];
+            if (!row.textContent.toLowerCase().includes(searchInput.value.toLowerCase())) {
+                row.style.display = 'none';
+            } else {
+                row.style.display = '';
+            }
+        }
+    });
+});
```
Comment (huongnt-2545) [inline air_quality_management/myapp/static/js/search.js]: e dùng jquery để cú pháp js ngắn gọn hơn nhé

**Comment by quandang246** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**